src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/aws/AwsConnector.java

```diff
@@ -91,56 +105,30 @@
 
 @Service
 public class AwsConnector implements CloudPlatformConnector {
-
+    private static final Logger LOGGER = LoggerFactory.getLogger(AwsConnector.class);
     private static final int MAX_POLLING_ATTEMPTS = 60;
     private static final int POLLING_INTERVAL = 5000;
     private static final int INFINITE_ATTEMPTS = -1;
-    private static final Logger LOGGER = LoggerFactory.getLogger(AwsConnector.class);
-
-    @Autowired
-    private AwsStackUtil awsStackUtil;
-
-    @Autowired
-    private Reactor reactor;
-
-    @Autowired
-    private ASGroupStatusCheckerTask asGroupStatusCheckerTask;
-
-    @Autowired
-    private CloudFormationTemplateBuilder cfTemplateBuilder;
-
-    @Autowired
-    private RetryingStackUpdater stackUpdater;
-
-    @Autowired
-    private CloudFormationStackUtil cfStackUtil;
-
-    @Autowired
-    private InstanceMetaDataRepository instanceMetaDataRepository;
-
-    @Autowired
-    private PollingService<AwsInstances> awsPollingService;
-
-    @Autowired
-    private PollingService<AutoScalingGroupReady> pollingService;
-
-    @Autowired
-    private PollingService<CloudFormationStackPollerContext> stackPollingService;
-
-    @Autowired
-    private ClusterRepository clusterRepository;
-
-    @Autowired
-    private UserDataBuilder userDataBuilder;
-
-    @Autowired
-    private StackRepository stackRepository;
-
-    @Autowired
-    private AwsInstanceStatusCheckerTask awsInstanceStatusCheckerTask;
-
-    @Autowired
-    private CloudFormationStackStatusChecker cloudFormationStackStatusChecker;
+    private static final String CLOUDBREAK_EBS_SNAPSHOT = "cloudbreak-ebs-snapshot";
+    private static final int SNAPSHOT_VOLUME_SIZE = 10;
+
+    @Autowired private AwsStackUtil awsStackUtil;
+    @Autowired private Reactor reactor;
+    @Autowired private ASGroupStatusCheckerTask asGroupStatusCheckerTask;
+    @Autowired private CloudFormationTemplateBuilder cfTemplateBuilder;
+    @Autowired private RetryingStackUpdater stackUpdater;
+    @Autowired private CloudFormationStackUtil cfStackUtil;
+    @Autowired private InstanceMetaDataRepository instanceMetaDataRepository;
+    @Autowired private PollingService<AwsInstances> awsPollingService;
+    @Autowired private PollingService<AutoScalingGroupReady> autoScalingGroupReadyPollingService;
+    @Autowired private PollingService<CloudFormationStackPollerObject> cloudFormationStackPollerObjectPollingService;
+    @Autowired private PollingService<EbsVolumeStatePollerObject> ebsVolumeStatePollerObjectPollingService;
```

Comment:
why is it's name so complicated? it can be as simple as `ebsVolumePoller`.
the same applies to `cloudFormationStackPollerObjectPollingService` that can be called `cloudFormationStackPoller`